packages/styles/accordion.css

```
@@ -0,0 +1,89 @@
+.Accordion__trigger,
+.Accordion > .ExpandCollapse__trigger {
+  background-color: var(--white);
+  padding: 12px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  border-radius: 3px;
+  border: 1px var(--gray-40) solid;
+  background-size: cover;
+  font-size: var(--text-size-small);
+  margin-top: 16px;
+  color: var(--gray-90);
+  text-decoration: underline solid var(--gray-90);
+}
+
+button.Accordion__trigger.expanded {
+  border-bottom-right-radius: 0px;
+  border-bottom-left-radius: 0px;
+  background-color: var(--gray-20);
+}
+
+button.Accordion__trigger:hover {
+  box-shadow: inset 3.5px 0px var(--accent-primary);
+  color: var(--accent-primary);
+  transition: all 0.2s ease-in-out;
+  text-decoration: underline solid var(--accent-primary);
+}
+
+button.Accordion__trigger:hover .Icon {
+  color: var(--gray-90);
+}
+
+.Accordion__panel {
+  background-color: var(--white);
+  color: var(--gray-90);
+  padding: var(--space-small);
+  border-left: 1px;
+  border-bottom: 1px;
+  border-right: 1px;
+  border-style: solid;
+  border-top: 0px;
+  border-color: var(--gray-40);
+  border-bottom-left-radius: 3px;
+  border-bottom-right-radius: 3px;
+  box-shadow: 0px 2px 3px 0px #00000015;
```

Nit: `rgba` is more widely supported. Also `0` values do not need a unit.

```suggestion
  box-shadow: 0 2px 3px 0 rgba(0, 0, 0, .15);
```